controllers/autoneg.go

```diff
@@ -283,6 +278,11 @@ func (b *ProdBackendController) ReconcileBackends(actual, intended AutonegStatus
 					newSvc.Backends = append(newSvc.Backends, &newBackend)
 				}
 			}
+			for beidx, be := range newSvc.Backends {
```

Comment:
I'm a bit worried about this - not all of the of the backends may be managed by autoneg.

Reply:
Isn't `newSvc` a combination of what is coming from upstream + what changed locally? If I'm not mistaken `CapacityScaler` will respect the value that was originally coming from the API, no?